rs/registry/admin/bin/main.rs

```diff
@@ -2253,14 +2254,21 @@ impl ProposalPayload<AddNodeOperatorPayload> for ProposeToAddNodeOperatorCmd {
             .map(|s| parse_rewardable_nodes(s))
             .unwrap_or_default();
 
+        let node_allowance = self.node_allowance.map(|allowance| {
+            if allowance != 0 {
+                eprintln!("WARNING: node allowance is deprecated and will be removed in the future. Overriding value to 0");
+            }
+            0
+        }).unwrap_or_default();
```

Comment:
Do not try to push forward, because then, you are telling the following lie to the user: I'm doing what you asked me to do. It does not matter that you print a warning: actions (e.g. pushing forward) speak louder than words. Instead, exit(1) (and explain!).

> Errors should never pass silently.
> --Zen of Python